itunes-connect.csv

```diff
@@ -0,0 +1,27 @@
+Name,Aware
+Privacy Policy URL,
+Bundle ID,com.github.josh.Aware
```

Comment:
Hoping to finalize this. Wondering if we should get a custom domain. I feel like that expense may end up eating into our nonexistent margins 😝